src/Cursor.js

```diff
@@ -37,8 +37,10 @@ function update(cmp, path, operation, nextValue) {
   var nextState;
 
   if (path.length > 0) {
+    var q = cmp._reactInternalInstance._pendingStateQueue;
```

Comment:
@meta-meta I'll merge this. But this looks not very future proof. Did you get any feel for what other alternatives there are? Maybe there is a better approach that react-cursor can take that might not need to inspect react internals now?